alerts/ssh_access_fromnagios.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+
+from lib.alerttask import AlertTask
+from mozdef_util.query_models import SearchQuery, TermMatch, TermsMatch, PhraseMatch
+
+
+class AlertSSHFromNagios(AlertTask):
```

Comment:
Since this alert isn't really specific to nagios, it's more of a "alert if any host in a list of configs, connects to some other host", should we name it more generically?

Reply:
Done